content/blog/2018/06/2018-05-14-simple-pull-request-plugin.adoc

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: "GSoC Project Intro: Simple pull request Plugin"
+tags:
+- gsoc2018
+- plugin
+author: abhishek_gautam
+---
+
+=== About me
+
+I am Abhishek Gautam, 3rd year student from Visvesvaraya National Institute of
+technology, India, Nagpur. I was a member of ACM Chapter and Google student developer club of his
+college. I am passianate about automation.
+
+=== Project Summary
+This is a GSoC 2018 project.
+
+This project aims to develop a Pull request Job Plugin. Users should be able to
+configure job type using YAML file which will be placed in root directory of the
+Git repository being the subject of the PR. Plugin should interact with various
+platfoms like Bitbucket, Github, Gitlab, etc whenever a pull request is created or updated.
+
+Detect the presence of certain types of the report based on a conventional location,
+and automatically publish them. If the reports are not in a conventional location,
+users could specify the location using the YAML file.
+
+==== Benefits to the community
+* Project administrators will be able to handle builds for pull requests more easily.
+* Build specifications for pull request can be written in a more readable format.
+* Build reports will be automatically published to Github, Bitbucket, ect.
+* Build status updates will be sent to git servers automatically.
+* Users will not have to deal with pipeline code.
+
```

Comment:
Would be nice to have a "Prior work" section here which briefly describes existing plugins and prototypes which support job definitions in YAML

Reply:
Done